README.md

```diff
@@ -53,55 +64,8 @@ print(f"The server has {status.players_online} players online and replied in {st
 
 ### Command Line Interface
 
-At present time, this only works with Java servers, Bedrock is not yet supported.
-```
-$ mcstatus
-Usage: mcstatus [OPTIONS] ADDRESS COMMAND [ARGS]...
-
-  mcstatus provides an easy way to query Minecraft servers for any
-  information they can expose. It provides three modes of access: query,
-  status, and ping.
-
-  Examples:
-
-  $ mcstatus example.org ping
-  21.120ms
-
-  $ mcstatus example.org:1234 ping
-  159.903ms
-
-  $ mcstatus example.org status
-  version: v1.8.8 (protocol 47)
-  description: "A Minecraft Server"
-  players: 1/20 ['Dinnerbone (61699b2e-d327-4a01-9f1e-0ea8c3f06bc6)']
-
-  $ mcstatus example.org query
-  host: 93.148.216.34:25565
-  software: v1.8.8 vanilla
-  plugins: []
-  motd: "A Minecraft Server"
-  players: 1/20 ['Dinnerbone (61699b2e-d327-4a01-9f1e-0ea8c3f06bc6)']
-
-Options:
-  -h, --help  Show this message and exit.
-
-Commands:
-  json    combination of several other commands with json formatting
-  ping    prints server latency
-  query   detailed server information
-  status  basic server information
-```
-
-## Installation
-
-Mcstatus is available on [PyPI](https://pypi.org/project/mcstatus/), and can be installed trivially with:
-
-```bash
-python3 -m pip install mcstatus
-```
-
-Alternatively, just clone this repo!
+This only works with Java servers; Bedrock is not yet supported. Use `mcstatus -h` to see helpful information on how to use this script.
 
 ## License
 
-Mcstatus is licensed under the Apache 2.0 license.
+Mcstatus is licensed under the Apache 2.0 license. See LICENSE for full text.
```

Comment:
```suggestion
Mcstatus is licensed under the Apache 2.0 license. See [LICENSE](./LICENSE) for full text.
```

Reply:
You know. I thought about this too and decided not to because I was too lazy to lookup how to make a link to a file on GitHub. Then I thought, even after I overcome that, how is this going to work on PyPI and Read the Docs. At that point, I decided to hell with that. Any ideas?

Reply:
Oh yeah, you're absolutely right I haven't thought about the external sites pulling our readme. Well, we could use a full link to the github file, but that's probably overkill here, alright let's leave it as-is.